recipes/gw/conda_build_config.yaml

```diff
@@ -2,3 +2,10 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "10.15"                # [osx and x86_64]
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.15"                # [osx and x86_64]
+
+c_stdlib_version:          # [unix]
+  - 2.28                   # [linux]
+
+docker_image:                                   # [os.environ.get("BUILD_PLATFORM", "").startswith("linux-")]
+  # Native builds
+  - quay.io/condaforge/linux-anvil-alma  # [os.environ.get("BUILD_PLATFORM") == "linux-64"]
```

Comment:
With our current setup this won't work. (We have the build image non configurable per recipe at Bioconda.)
I'd advice against this either way, though since, e.g., requiring glibc>=2.28 will go against our compatibility goals.

Reply:
Ok, thanks. I will remove this step then